wd_core.au3

```diff
@@ -1013,7 +1014,11 @@ Func _WD_Cookies($sSession, $sCommand, $sOption = Default)
 			$iErr = @error
 
 		Case 'delete'
-			$sResponse = __WD_Delete($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/cookie/" & $sOption)
+			$sResponse = __WD_Delete($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/cookie" & ($sOption <> '') ? "/" & $sOption : "")
+			$iErr = @error
+
+		Case 'deleteall'
+			$sResponse = __WD_Delete($_WD_BASE_URL & ":" & $_WD_PORT & "/session/" & $sSession & "/cookie")
 			$iErr = @error

```

Comment:
Should these be combined into a single `Case` entry?

Reply:
No because when some body use `deleteall` and also provide `$sOption` then there will issue and confusion.
Let it be like it is.

Reply:
Understood. However, nothing prevents them from doing that (passing a value in $sOption) in the current implementation of DeleteAll.

Reply:
Show your code proposal.

Reply:
ah... ok.

Reply:
take a look at this two comimts:

[Case 'delete', 'deleteall' + parameters validation](https://github.com/Danp2/au3WebDriver/pull/195/commits/d652e2f47ecf94b1dd9bad20836b0aed4eede5ed)
[d652e2f](https://github.com/Danp2/au3WebDriver/pull/195/commits/d652e2f47ecf94b1dd9bad20836b0aed4eede5ed)

[$_WD_ERROR_InvalidArgue](https://github.com/Danp2/au3WebDriver/pull/195/commits/7697cd3128f266a57282e020a0cd6718be2abdca)